docs/_quarto.yml

```diff
@@ -95,6 +95,7 @@ website:
       - sidebar:reference
     right:
       - posts.qmd
+      - sidebar:presentations
```

Comment:
might want to change this to a landing page where we can link to the presentations + recordings and perhaps where relevant (/perhaps other prose, like "we gave this talk at this linkedin meetup, it wasn't recorded, but here are the slides")

Reply:
Yep, turned into into a page.